workspace-manager-clienttests/src/main/java/scripts/disruptivescripts/DeleteInitialPods.java

```diff
@@ -0,0 +1,75 @@
+package scripts.disruptivescripts;
+
+import bio.terra.testrunner.common.utils.KubernetesClientUtils;
+import bio.terra.testrunner.runner.DisruptiveScript;
+import bio.terra.testrunner.runner.TestRunner;
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.models.V1Deployment;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+public class DeleteInitialPods extends DisruptiveScript {
+    private static final Logger logger = LoggerFactory.getLogger(DeleteInitialPods.class);
+
+    public DeleteInitialPods() {
+        super();
+        manipulatesKubernetes = true;
+    }
+
+    protected static final int SECONDS_TO_WAIT_BEFORE_STARTING_DISRUPT = 15;
+
+    @Override
+    public void setParameters(List<String> parameters) {
+
+        if (parameters == null || parameters.size() == 0) {
+            return;
+        }
+    }
+
+    @Override
+    public void disrupt(List<TestUserSpecification> testUsers) throws Exception {
+        // give user journey threads time to get started before disruption
+        TimeUnit.SECONDS.sleep(SECONDS_TO_WAIT_BEFORE_STARTING_DISRUPT);
+
+        logger.info("Starting disruption - all initially created api pods will be deleted, one by one.");
+
+        String componentLabelKey = "app.kubernetes.io/component";
```

Comment:
I have added this comment to [Test Runner Enhancement](https://docs.google.com/document/d/1gYqlJTpC4C4S1K0pYikeNJx-HWNZAZJYybinmuqcSTI/edit?usp=sharing) doc as a future enhancement.

Reply:
I think it would be good to replace `componentLabelKey` here with `KubernetesClientUtils.COMPONENT_LABEL` instead.

If you want to handle adding a new method that doesn't require passing in this value, as a separate ticket, that seems fine to me.

Reply:
This has been address by passing in the default values from application specification.